### DISCUSSION TITLE
Bug when exporting (v. 3.1.3 macOS Catalina)

### DISCUSSION THREAD

**Discussion by acalvoe** [audacity/audacity#3040] [Development]
Hello,

I've found a bug when exporting to any format. Although my project with some tracks and clips sounds OK in the aup3 format and saves correctly any changes I make, the exported output (MP3, WAV, etc.) has minor mistakes when two clips in different tracks overlap and sound at the same time (think of editing a song for example), and that makes it impossible to get an accurate audio output and indeed renders the program of no use for a decent editing. The mistakes are small errors in positioning, at least (I think maybe volume too in the ares they overlap).

It's a bug that I guess should be easily fixed and that's what I ask for.

Thanks.

**Comment by LWinterberg** [comment]
can you record a little screen capture showing what you're doing and what exactly is happening?
